app/src/server.js

```diff
@@ -59,7 +64,17 @@ app.get(
   }
 );
 
-const server = app.listen(port, function() {
+// Setup and start the HTTPS server.
+
+const key = fs.readFileSync("./certs/key.pem", "utf-8");
```

Comment:
these paths should be configurable via ENV vars (like `PORT`) - logging what's going on would be nice (i.e. loading certs and paths)

Reply:
I'll open a story to introduce logging